doc/ampi/manual.rst

```diff
@@ -747,13 +894,38 @@ different schemes.
    FS-Globals           Yes Yes    Yes    No   Maybe   Yes   Yes
    ==================== === ====== ====== ==== ======= ===== =====
```

Comment:
The table could use some improvements:

- x86, x86_64 -> Linux x86 (32/64 bit) (same for PPC, ARM7)
- No -> Not supported
- Maybe -> Not implemented (yet)
- Transformation -> Source-to-Source Transformation
- order in the same way as they are discussed in the manual

Reply:
I added a Linux column and moved the x86 architectures to be next to the others. Specifying OS and arch in the same column would be too verbose.

"No" and "Maybe" are already explained in the table's key, and changing them would blow up the size.

Changed "Transformation" to "Manual Change" and reordered the rows to match the document's text.

Reply:
> I added a Linux column and moved the x86 architectures to be next to the others. Specifying OS and arch in the same column would be too verbose.
> 
> "No" and "Maybe" are already explained in the table's key, and changing them would blow up the size.

+1.

> Changed "Transformation" to "Manual Change" and reordered the rows to match the document's text.

My concern is that the connection to the manual section where this transformation is explained isn't too clear. Maybe change the table entry and subsection title to "Source code change"?

Reply:
Changed the Manual Change table entry and section title to Manual Code Editing.